smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/scheduler/CompressionScheduler.java

```diff
@@ -163,7 +163,7 @@ public boolean supportDecompression(String path) throws MetaStoreException, IOEx
     if (fileState instanceof CompressionFileState) {
       return true;
     }
-    LOG.warn("A compressed file path should be given!");
+    LOG.debug("A compressed file path should be given!");
```

Comment:
In my opinion, "warn" or "error" looks better to deliver critical info to user. Using "debug" will hide the message since generally "debug" log level is generally not used in product environment. Am I missing something?

Reply:
This log info will be reported every time the rule is checked for decompression. I think this will mislead users about the effect of the rule.
![image](https://user-images.githubusercontent.com/32612402/104549351-b00d0780-566d-11eb-97cb-90248529e8ad.png)